src/cadenza.js

```diff
@@ -671,6 +700,7 @@ function createParams({
     ...(locationFinder && { locationFinder }),
     ...(mapExtent && { mapExtent: mapExtent.join() }),
     ...(mediaType && { mediaType }),
+    ...(parts && { parts: parts.toString() }),
```

Comment:
I'd rather always use the same method of joining an array, i.e. .join() (like above).